.storybook-preact/head.html

```diff
@@ -0,0 +1,5 @@
+<style>
+  html, body, #root {
+    height: 100%;
+  }
+</style>
```

Comment:
Are the files in `.storybook-preact` the same as in `.storybook`, aside from the Webpack aliases?

If so, maybe there's a way we can use a single `.storybook` directory, but pass in some kind of argument from the CLI specifying which React lib to use:

```js
// Just an example, not sure how we'd get this value:
const reactLib = process.argv[2];

const aliases = {
  preact: {
      "react": "preact-compat",
      "react-dom": "preact-compat",
      "react-impl": "preact"
  },
  // Also not sure if a React alias is required (see further comment), but assuming it is:
  react: {...},
};

module.exports = {
  resolve: {
    alias: aliases[reactLib],
  },
};
```